bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/AVMFritzHandlerFactory.java

```diff
@@ -84,6 +78,9 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
             return new DeviceHandler(thing);
         } else if (GROUP_HEATING_THING_TYPE.equals(thingTypeUID)) {
             return new AVMFritzHeatingGroupHandler(thing);
+        } else if (SUPPORTED_LIGHTING_THING_TYPES.contains(thingTypeUID)) {
+            logger.info("WOOHOOOOOO!");
```

Comment:
Hm ... I never expect this to be happening as long as the `DECT500_THING_TYPE` is part of the set `SUPPORTED_DEVICE_THING_TYPES_UIDS`. You should remove it from that set.

Reply:
If i put it off this set, then auto discovery won't detect the DECT500 anymore. 

AVMFritzDiscoveryService:
`   @Override
    public void onDeviceAdded(AVMFritzBaseModel device) {
        ThingTypeUID thingTypeUID = new ThingTypeUID(BINDING_ID, bridgeHandler.getThingTypeId(device));
        if (getSupportedThingTypes().contains(thingTypeUID)) {
            ThingUID thingUID = new ThingUID(thingTypeUID, bridgeHandler.getThing().getUID(),
                    bridgeHandler.getThingName(device));
            onDeviceAddedInternal(thingUID, device);
        } else {
            logger.debug("Discovered unsupported device: {}", device);
        }
    }`
    
I just move the lines up a bit, so it will execute the lines
'      } else if (SUPPORTED_LIGHTING_THING_TYPES.contains(thingTypeUID)) {
            logger.info("WOOHOOOOOO!");' 
before the lookup for SUPPORTED_DEVICE_THING_TYPES_UIDS

Reply:
just realized, the avm autodiscovery has it's own set of discoverable things.